src/renderer/apis/settings.ts

```diff
@@ -1,4 +1,5 @@
 import type { Jsonifiable } from "type-fest";
+import { React } from "@common";
```

Comment:
I don't think we should be involving webpack modules in our APIs. Webpack modules can certainly *use* the APIs, but they should not be *part of* the APIs.